packages/dashboard/src/hooks/useSnackbarCallback.ts

```diff
@@ -0,0 +1,38 @@
+import { useDashboardI18N } from '../locales'
+import { useSnackbar } from 'notistack'
+import { useCallback } from 'react'
+
+export function useSnackbarCallback<P extends (...args: any[]) => Promise<T>, T>({
```

Comment:
Is it also used in the main package? Can we move it to `maskbook-shared` package?

Reply:
cc @nuanyang233